tests/ICU4N.Tests/Text/RuleBasedBreakIteratorTest.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using ICU4N.Globalization;
+using NUnit.Framework;
+
+namespace ICU4N.Text;
+
+public class RuleBasedBreakIteratorTest
+{
+    [Test]
+    public void HeavyParallelLoad_WithRandomStrings_ShouldNotThrow_Issue95()
```

Comment:
Thanks for finding that. I backported the test and it would not fail for me with the DequeI change reverted, even if I greatly increase the iteration count, so I added my test as well to this file. But I still think it's a good test to keep so I kept both.